packages/flutter/test/cupertino/text_field_test.dart

```diff
@@ -173,7 +173,7 @@ void main() {
 
   setUp(() async {
     EditableText.debugDeterministicCursor = false;
-    // Fill the clipboard so that the PASTE option is available in the text
+    // Fill the clipboard so that the Paste option is available in the text
```

Comment:
nit: the cupertino localization file is not updated so this is probably not needed?

Reply:
Good point.  I'm going to keep it as Paste for consistency, and because the iOS text selection menu isn't all caps either. It probably should have been Paste before this PR.